.github/workflows/grafana-foundation-sdk-diff-preview.yaml

```diff
@@ -29,32 +29,48 @@ jobs:
           enable-cache: 'true'
           devbox-version: ${{ env.DEVBOX_VERSION }}
 
-      - name: Dry-run release
+      - name: Dry-run release with current branch
         shell: bash
         run: |
           git config --global user.email "cog-ci@grafana.com"
           git config --global user.name "cog - CI"
-          
-          devbox run ./scripts/release-version.sh next next
+
+          devbox run ./scripts/release-version.sh next
         env:
-          FOUNDATION_SDK_PATH: ./grafana-foundation-sdk
+          FOUNDATION_SDK_PATH: ../grafana-foundation-sdk-current
+          FOUNDATION_SDK_REPO: https://github.com/grafana/grafana-foundation-sdk.git
+          LOG_LEVEL: '7' # debug
+          GOGC: 'off'
+
+      - name: Checkout main branch
+        uses: actions/checkout@v4
+        with:
+          ref: main
+
+      - name: Dry-run release with main branch
+        shell: bash
+        run: |
+          git config --global user.email "cog-ci@grafana.com"
+          git config --global user.name "cog - CI"
+
+          devbox run ./scripts/release-version.sh next
+        env:
+          FOUNDATION_SDK_PATH: ../grafana-foundation-sdk-main
           FOUNDATION_SDK_REPO: https://github.com/grafana/grafana-foundation-sdk.git
           LOG_LEVEL: '7' # debug
           GOGC: 'off'
 
       - name: Preview diff
         run: |
-          cd ./grafana-foundation-sdk
-          
           echo -e '<!-- grafana-foundation-sdk-diff-preview-marker -->\n\n' >> preview.md          
```

Comment:
Suggest below shell construct, mainly to save the many `echo -e ...`, into only ~3 blocks of code (note the 1st single-quoted `'EOF'` is important, to avoid shell expansion e.g. for the backquotes strings):

```shell
cat <<'EOF' > preview.md
**Note:** the diff includes only changes introduced by this PR. Changes on `main` are excluded.

<details>
<summary>
### 🔎 Changes to `grafana-foundation-sdk@next+cog-v0.0.x`
</summary>
</details>

```patch
EOF

diff --new-file --unidirectional-new-file --color=never --unified --recursive --exclude='README.md' --exclude='.git' --exclude='gradle.properties' --exclude='pyproject.toml' --exclude='package.json' --exclude='Installing.md' ../grafana-foundation-sdk-main/ ../grafana-foundation-sdk-current/ | tee -a preview.md

echo -e '```\n' >> preview.md
```

NOTE btw that by having a clearer "free-form" multi-line block of text, we can now spot the pre-existing missing `</details>` there ;)

Reply:
Deal, let's go with the EOF syntax! 

Note: `</details>` wasn't missing though :thinking: